.editorconfig

```diff
@@ -8,3 +8,6 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.html]
+indent_size = 4
```

Comment:
I added this to make my editor happy, but happy to drop this change.

Reply:
You must have a wide screen 😅 .

Reply:
Not so much, but that's in line with what's currently used in the Angular html files.

I'm not wedded to this 😉